### ISSUE TITLE
Example In Readme and In Docs, calls itself logistic regression, but is actually softmax classification?

### ISSUE THREAD

**Issue by oxinabox** [malmaud/TensorFlow.jl#98]
In  the readme, and in the [docs](https://malmaud.github.io/tfdocs/logistic/)
it is marked as being logistic regression.

But it looks to me like it is doing classification, using a softmax layer, with cross-entropy loss.
Am I missing something?

**Comment by oxinabox** [comment]
Apparently I am.
I'm used to using Logistric regression as a term solely as for binary classification.
[Wikipedia tells me](https://en.wikipedia.org/wiki/Logistic_regression), the the term is also used more generally, for categorical classification
